src/app/mauro/sort.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
+
+/*
+Copyright 2022-2023 University of Oxford
+and Health and Social Care Information Centre, also known as NHS Digital
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+@Injectable({
+  providedIn: 'root',
+})
+export class SortService {
+  ascString(a: string, b: string): number {
```

Comment:
These two functions look a bit too simple to include in an injectable service. Have you considered just making them pure TypeScript functions?

Reply:
I've changed to a static class, but I'm not 100% sure that is what was intended with this change.